ci/Jenkinsfiles/build.groovy

```diff
@@ -554,16 +559,14 @@ pipeline {
     stage('Run "dev" functional tests') {
       steps {
         setGitHubBuildStatus('ftests/dev', 'Functional tests - dev environment', 'PENDING')
-        retry(2) {
-          container('maven') {
-            echo """
-            ----------------------------------------
-            Run "dev" functional tests
-            ----------------------------------------"""
-            runFunctionalTests('ftests')
-          }
+        container('maven') {
+          echo """
+          ----------------------------------------
+          Run "dev" functional tests
+          ----------------------------------------"""
+          runFunctionalTests('ftests')
         }
-        findText regexp: ".*ERROR.*", fileSet: "ftests/**/log/server.log", unstableIfFound: true
+        findText regexp: ".*ERROR.*", fileSet: "ftests/**/log/server.log"
```

Comment:
Does this means that we now accept errors in server log?

Reply:
Nop, it means it fails the build, so a failure status, if there are errors in logs.